tools/odbc/supported_tests

```diff
@@ -3,4 +3,5 @@ boolsaschar-test
 connect-test
 numeric-test
 quotes-test
+result-conversions-test
```

Comment:
Could you add the different timestamp types to the tests? e.g. the following SQL statements:

```sql
select timestamp '2021-07-15 12:30:00'::timestamp_s;   -- TIMESTAMP_SEC
select timestamp '2021-07-15 12:30:00'::timestamp_ms;  -- TIMESTAMP_MS
select timestamp '2021-07-15 12:30:00';                -- TIMESTAMP, the default type
select timestamp '2021-07-15 12:30:00'::timestamp_ns;  -- TIMESTAMP_NS
```

Then try to read the TIME, DATE and TIMESTAMP of each of these.